test/spec/features/copy-paste/BpmnCopyPasteSpec.js

```diff
@@ -71,6 +71,75 @@ describe('features/copy-paste', function() {
       }));
 
 
+      describe('should copy boundary events without host', function() {
+
+        it('should copy/paste', inject(function(elementRegistry, canvas, copyPaste) {
+
+          // given
+          var boundaryEvent = elementRegistry.get('BoundaryEvent_1'),
+              rootElement = canvas.getRootElement();
+
+          // when
+          copyPaste.copy(boundaryEvent);
+
+          var copiedElements = copyPaste.paste({
+            element: rootElement,
+            point: {
+              x: 1000,
+              y: 1000
+            }
+          });
+
+          // then
+          expect(rootElement.children).to.have.length(2);
+
+          expect(copiedElements).to.have.length(1);
+
+          expect(copiedElements[0].type).to.equal('bpmn:IntermediateCatchEvent');
+
+          expect(copiedElements[0].id).not.to.equal(boundaryEvent.id);
+        }));
+
+
+        it('should copy/paste and reattach', inject(function(elementRegistry, canvas, copyPaste) {
+
+          // given
+          var boundaryEvent = elementRegistry.get('BoundaryEvent_1'),
+              task = elementRegistry.get('Task_1'),
+              rootElement = canvas.getRootElement();
+
+          // when
+          copyPaste.copy(boundaryEvent);
+
+          var copiedElement = copyPaste.paste({
+            element: rootElement,
+            point: {
+              x: 1000,
+              y: 1000
+            }
+          })[0];
+
+          copyPaste.copy(copiedElement);
+
+          var attachedBoundaryEvent = copyPaste.paste({
+            element: task,
+            point: {
+              x: task.x,
+              y: task.y
+            },
+            hints: {
+              attach: 'attach'
+            }
+          })[0].businessObject;
+
+          // then
+          expect(attachedBoundaryEvent.attachedToRef).to.eql(task.businessObject);
```

Comment:
Please keep the tests consistent. In the first test, you're checking the element's type, in the second you're checking the `attachedToRef` property.

```javascript
it('should copy and paste boundary event', function() {
  ...
  expect(businessObject.$type).to.equal('bpmn:IntermediateCatchEvent');
  expect(businessObject.attachedToRef).not.to.exist;
});

it('should copy and paste and reattach boundary event', function() {
  ...
  expect(businessObject.$type).to.equal('bpmn:BoundaryEvent');
  expect(businessObject.attachedToRef).to.exist;
});
```